packages/shared-config/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./file-manager";
```

Comment:
I'm wondering whether we should include `export * as filemanager from "./file-manager";` in index.ts, or leave it out, so that imports would instead use: `import { ... } from '@orcabus/platform-cdk-constructs/shared-config/file-manager';`. Thinking of avoiding exporting everything as part of the shared config.

Reply:
Yes, good idea.